org.dupot.beatandmatchtopass.yml

```diff
@@ -0,0 +1,24 @@
+id: org.dupot.beatandmatchtopass
+runtime: org.freedesktop.Platform
+runtime-version: "21.08"
+sdk: org.freedesktop.Sdk
+command: beatmatchtopass.x86_64
+finish-args:
+  - --share=ipc
+  - --socket=pulseaudio
+  - --socket=x11
+  - --device=dri
+  - --persist=.local/share/godot/app_userdata/BeatMatchtoPass
+
+modules:
+  - name: BeatMatchtoPass
+    buildsystem: simple
+    build-commands:
+      - install -Dm555 export/linux/beatmatchtopass.x86_64 /app/bin/beatmatchtopass.x86_64
+      - install -Dm644 export/linux/flatpack/512x512.png /app/share/icons/hicolor/512x512/apps/org.dupot.beatmatchtopass.png
+      - install -Dm644 export/linux/flatpack/org.dupot.beatmatchtopass.desktop /app/share/applications/org.dupot.beatmatchtopass.desktop
+      - install -Dm644 export/linux/flatpack/org.dupot.beatmatchtopass.appdata.xml /app/share/metainfo/org.dupot.beatmatchtopass.appdata.xml
```

Comment:
https://github.com/imikado/dupotBeatAndMatchToPass/blob/67167b59b4bef2ebe6246400471df0949146bb58/export/linux/flatpack/org.dupot.beatmatchtopass.appdata.xml#L16-L34 Pull screenshots from a tag/commit not main

Reply:
fixed

Reply:
how?

Reply:
i push modifications for this file, i replace absolute by $FLATPAK_DEST/share...

Reply:
that's not what was asked here. You didn't address the `Pull screenshots from a tag/commit not main`

Still pull from main

![image](https://github.com/flathub/flathub/assets/114441/52516dae-c246-4f41-9351-bba85ac866f6)

Reply:
i just fixed it

Reply:
(i just commit)